web/themes/new_weather_theme/templates/block/block--weathergov-local-alerts.html.twig

```diff
@@ -56,69 +56,88 @@
+                      <li class="">{{ city }}</li>
+                    {% endfor %}
+                    </ul>
+                    </wx-alert-cities>
+                  {% endif %}
 
-            <wx-alert-map class="display-block grid-col-12 wx-radar-container wx-isolation-isolate" data-geo-json="{{ alert.geometry | json_encode | url_encode }}" data-alert-id="{{ alert.alertId}}" data-lat="{{ weather.point.lat }}" data-lon="{{ weather.point.lon }}" data-alert-name="{{ alert.event }}">
-              <div id="wx_alert_map_{{ alert.alertId }}" class="height-full"></div>
-              <div class="wx_alert_map_legend margin-top-3 display-flex flex-align-center">
-                <div class="wx_alert_map_legend_impact_area display-block width-3 height-3 border-2px margin-right-1"></div><span>Impacted Area</span>
+                {#
+                  If the alert doesn't have that location information in the description
+                  text, we can use the list of impacted areas provided by AWIPS. This
+                  list is based on the alert polygon drawn by a forecaster, and then
+                  AWIPS identifies the areas.
+                #}
+                {% else %}
+                  <ul class="usa-list {%- if alert.areaDesc | length > 7 %} wx-col-2 {%- endif -%}">
+                    {%  for areaName in alert.areaDesc %}
+                      <li class="">{{ areaName }}</li>
+                    {% endfor %}
+                  </ul>
+                {% endif %}
               </div>
-            </wx-alert-map>
+
+              <div class="grid-col-12 tablet-lg:grid-col-6 margin-top-3">
+              <wx-alert-map class="display-block grid-col-12 wx-radar-container wx-isolation-isolate" data-geo-json="{{ alert.geometry | json_encode | url_encode }}" data-alert-id="{{ alert.alertId}}" data-lat="{{ weather.point.lat }}" data-lon="{{ weather.point.lon }}" data-alert-name="{{ alert.event }}">
+                <div id="wx_alert_map_{{ alert.alertId }}" class="height-full"></div>
+                <div class="wx_alert_map_legend margin-top-3 display-flex flex-align-center">
+                  <div class="wx_alert_map_legend_impact_area display-block width-3 height-3 border-2px margin-right-1"></div><span>Impacted Area</span>
+                </div>
+              </wx-alert-map>
+            </div>
+          </div>
+        </div>
+
+        <hr class="border-base-light">
 
         {{ drupal_block("weathergov_dynamic_safety_information", { weather_event: alert.event }) }}
 
         </div>
       </div>
       {% endfor %}
       </wx-alerts>
+
+      <script type="text/javascript">
+        {#
+          We can't set the link classes in Drupal, so do it on the browser side,
+          immediately after rendering, before the user actually sees it.
+        #}
+        Array.from(document.querySelectorAll(".wx-safety-tips a")).forEach(node => node.classList.add("usa-link"));
```

Comment:
It's not ideal, but since the links are embedded in content managed by Drupal, I didn't know how else to do it. I do want to fiddle with it some more to see if I can find a server-side way.

Reply:
For future reference, `.usa-link` is defined with an `@include`, and we can use the same one to get the same styling:

```css
.usa-prose.wx-safety-tips a {
  @include typeset-link;
}
```

Reply:
Nice!!